### PR TITLE
update schema and .gitconfig

### PR DESCRIPTION
**Pull request by KanikaWadhawan**
# Update Prisma Schema 

- Update User
- Add Event 
- Add Campaign
- Add Comment

### CODE REVIEW

File: .gitconfig [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  whitespace = red reverse
+[color]
+    ui = auto
+[alias]
+    st = status
+    ci = commit --verbose
+    co = checkout
+    di = diff -w
+    dc = diff --cached -w
+    noff = merge --no-ff
```
Comment (siddharthparmar7) [inline .gitconfig]: This file should be in the $HOME dir. Please refer to https://github.com/redacademy/wr-fall-2019/wiki/Git-Merge-Workflow

File: server/prisma/datamodel.prisma [hunk 2]
```diff
@@ -3,9 +3,31 @@ type User {
...
 
-type Post {
+type Event{
+   id: ID! @id
+   title: String! @unique
+   createdAt: DateTime! @createdAt
+   location: String!
+   image: String!
+   description: String!
+   comments: [Comment!]!
```
Comment (siddharthparmar7) [inline server/prisma/datamodel.prisma:16]: Do you need a field for Campaign here?

File: .gitconfig (hunk 1 above)
Comment (siddharthparmar7) [reply .gitconfig]: I think you are not able to use this aliases at the moment. Please check.

File: server/prisma/datamodel.prisma (hunk 2 above)
Comment (siddharthparmar7) [inline server/prisma/datamodel.prisma:16]: Does your user need to related to the events?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

File: server/prisma/datamodel.prisma (hunk 2 above)
Comment (KanikaWadhawan) [reply server/prisma/datamodel.prisma:16]: Yes, the user has MyCampaign screen which is a list of their campaigns.

Comment (KanikaWadhawan) [reply server/prisma/datamodel.prisma:16]: No, the user has no relation with the events. But, has direct relation with the campaign. 
Whether the user is signed in or not, the events section on the Campaign appears the same way.

File: .gitconfig (hunk 1 above)
Comment (KanikaWadhawan) [reply .gitconfig]: Sure, thank you.

File: server/prisma/datamodel.prisma (hunk 2 above)
Comment (siddharthparmar7) [reply server/prisma/datamodel.prisma:16]: ok thanks.

Comment (siddharthparmar7) [reply server/prisma/datamodel.prisma:16]: If you need to see what campaign this event belongs to then please add a field for a campaign here. Thank you.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**